plugins/woocommerce-blocks/assets/js/blocks/product-collection/style.scss

```diff
@@ -0,0 +1,48 @@
+// This animation will be shown when user change the pages in the product collection block
+.wc-block-product-collection__pagination-animation {
+	position: fixed;
+	top: 0;
+	left: 0;
+	margin: 0;
+	padding: 0;
+	width: 100vw;
+	max-width: 100vw;
+	height: 4px;
+	background-color: var(--wp--preset--color--primary, #000);
+	opacity: 0;
```

Comment:
What do you think about adding `pointer-events: none` to this element? It's always there in a DOM and with some minimalistic design it may be covering some interactive elements (like in the screenshot below, where I intentionally made it green). I know it's super edge case for someone to click in the top 4px but doesn't hurt to add it as `pointer-events: none` allows passing clicks to the interactive elements "below".

<img width="847" alt="image" src="https://github.com/woocommerce/woocommerce/assets/20098064/df4b7aac-9d98-44af-bf70-2aea674e2d1f">

Reply:
Good catch! I have made the changes in https://github.com/woocommerce/woocommerce/pull/44571/commits/22603a8673517036b1de7b744b54a425fc68fab1 🙌🏻

Looks like this could be improved in Query Loop animation too 🙂